### DISCUSSION TITLE
Any idea how to load this into database, ideally PostgreSQL?

### DISCUSSION THREAD

**Discussion by MironAtHome** [kblin/ncbi-genome-download#189] [Q&A]
I am keen to learn how someone stores this data in database for post processing

**Comment by kblin** [comment]
What kind of data? The actual sequence data you downloaded? The metadata on which files were downloaded?

**Comment by MironAtHome** [comment]
Thank you for your kind following up.
I am not quite yet in the main on what those files actually are.
I thought there is code that knows how to parse / handle these files, from which I can learn both, what those files deep down are, and how to manipulate the data.
In my mind DNA/RNA sequence, aking back to biology classes, was a sequence of 4 digits. That's what I was expecting to find. And this is where my main question actually is.
There is a great deal of variety of ways one could store such a 4 letter language.
I was particularly shy going for storing actual letters, since letters are very obtrusive things, that have a lot of connotation and a lot of background.
What I was sort of expecting is metadata that would help me structure helix out of succeeding lines of letters. Specifically, how the helix forming intra sequences develop within the succeeding singularities. Instead I found a lot of sharding, based on experience, exploration and experiment based grouping of symbols. Which is extremely confusing and arbitrary. So, when you ask me about the "files" and such, please be gentle :)
And yes, any code that understands these groupings, showing how to interpret this data, would be absolutely helpful and at this moment essential.
And yes, I already found fasta, thank you Universities and fasta format creators! But any additional information would be very - very - very - very helpful.
